src/System.Security.Cryptography.RSA/src/System/Security/Cryptography/SafeCryptoHandles.cs

```diff
@@ -247,7 +266,8 @@ internal static SafeHashHandle InvalidHandle
         [SecurityCritical]
         protected override bool ReleaseHandle()
         {
-            //FreeHash(handle);
+            bool successfullyFreed = CapiHelper.CryptDestroyHash(handle);
+            Debug.Assert(successfullyFreed);
```

Comment:
In all of these cases, should we be returning successfullyFreed instead of the hardcoded true?